### PR TITLE
refactor(migrations): refactor firewall rules for migrations in delivery stage

### PR DESCRIPTION
**Pull request by luizjhonata**
## 🚦 Quality checklist
* [x]  Did you add the changes in the `CHANGELOG.md`?
* [x]  Did you run all the code checks? (`go test`)
* [x]  Are the tests passing?

### CODE REVIEW

File: azure-devops/golang/stages/40-delivery/arm.yaml [hunk 1]
```diff
@@ -47,34 +47,47 @@ stages:
...
+              inlineScript: |
+                # TODO: move this "IF" to a file and (or) use an Azure resource to create a condition
+                if grep -r 'goose' db/*.sql; then
+                  PIPELINE_IP=$(curl -s ifconfig.me/ip)
+                  PIPELINE_FIREWALL_NAME="PipelineFirewall_$(date +"%Y%m%d%H%M%S")"
+                  RESOURCE_GROUP_NAME=$(environment)-rg-$(functionAppName)-$(location)
+                  DATABASE_SERVER_NAME=$(OUTPUTS.databaseServername.value)
+
+                # TODO: create firewall for others databases 
+                  az --version
```
Comment (rios0rios0) [inline azure-devops/golang/stages/40-delivery/arm.yaml:65]: Do we need this line here?

**Review by rios0rios0 — CHANGES_REQUESTED** [review]
Please, look at the threads.

File: azure-devops/golang/stages/40-delivery/arm.yaml (hunk 1 above)
Comment (luizjhonata) [reply azure-devops/golang/stages/40-delivery/arm.yaml:65]: Yes, we need.

**Comment by luizjhonata** [comment]
> Please, look at the threads.

Corrected.

**Review by rios0rios0 — APPROVED** [review]

**Pull request merged**